.github/workflows/test_ledgers.yml

```diff
@@ -0,0 +1,33 @@
+name: Replay Ledgers
+on:
+  workflow_call:
+  workflow_dispatch:
+  pull_request:
+    types: [synchronize, opened, reopened, ready_for_review]
+  merge_group:
+jobs:
+  ledger-replay:
+    runs-on: private
+    if: github.event.pull_request.draft == false
+    env:
+      CC: gcc
+      EXTRAS: no-solana
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - uses: ./.github/actions/deps
+      - uses: ./.github/actions/hugepages
+        with:
+          count_gigantic: 300
+          count_huge: 1000 # TODO: this is required until we can handle anonymouse workspaces and loose huge pages in fddev
+
+      - name: build
+        run: |
+          make -j bin
+      - name: test replay ledgers
+        run: |
+          sudo prlimit --pid=$$ --nofile=1048576
+          sudo prlimit --pid=$$ --memlock=unlimited
+          python3.11 ./src/flamenco/runtime/tests/run_ledger_tests_all.py ./src/flamenco/runtime/tests/run_ledger_tests_all.txt
```

Comment:
Could we maybe use `make run-runtime-test` instead?

We have https://github.com/firedancer-io/firedancer/blob/main/src/flamenco/runtime/tests/Local.mk which has a list of ledger tests in, but now we have 2 lists of ledger tests which could be confusing.

Would be great if we had an authoritative list of the ledger tests.

Thanks so much :)